### PR TITLE
Create installLoadTestServer.rst

### PR DESCRIPTION
**Pull request by seenthattinker**
This is a rewrite of the jinstall markdown file. I switched to rst as it will go to the public facing docs site.

### CODE REVIEW

File: installLoadTestServer.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
+===========================
+Install Load Test  
+===========================
+
+
+The *loadtest* is installed atop an existing Mattermost installation. If you do not have Mattermost installed along with its ancillary componets then the installations in  must be completed.
```
Comment (crspeller) [inline installLoadTestServer.rst]: No it's not. It should be installed on a separate machine.

File: installLoadTestServer.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Load Test Install
+==========================================
+
+The following steps define the *loadtest* install.
+
+
+1. Install the build-essential package.
+
+    ``sudo apt-get install build-essential``
```
Comment (crspeller) [inline installLoadTestServer.rst:24]: Pretty sure macs don't have `apt`

File: installLoadTestServer.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    
+Linux
+--------------
+
+1. Open a ``Terminal`` and download the Go 1.8 binary file for Linux with the following command:
```
Comment (crspeller) [inline installLoadTestServer.rst:29]: We don't want to keep updating this for the latest go. Step 1 can just be install golang: https://golang.org/dl/

File: installLoadTestServer.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+    ``wget https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz``
+
+#. Install the Go 1.8 binary file for Linux with the following command:
+
+      ``sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz``
+
+#. Modify permissions on ``/usr/local/go`` by replacing {user} and {group} with the user and group to which you are logged in. Use the following command:
```
Comment (crspeller) [inline installLoadTestServer.rst:37]: Wah? Why?

File: installLoadTestServer.rst [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+#. Reload your bash configuration with the following command:
+
+    ``source ~/.bashrc``.
+    
+Install Glide on Linux and Mac
+--------------------------------------------------
+
+#. Install Glide with the following command:
+
+    ``curl https://glide.sh/get | sh``
```
Comment (crspeller) [inline installLoadTestServer.rst:71]: Loadtests don't use glide anymore. They use dep with the dependencies in the vendor directory. So no need for user side dependency management.

**Review by crspeller — CHANGES_REQUESTED** [review]
You should also meantion that you can just download the latest binary release from https://releases.mattermost.com/mattermost-load-test/mattermost-load-test.tar.gz

File: installLoadTestServer.rst (hunk 2 above)
Comment (seenthattinker) [reply installLoadTestServer.rst:24]: https://support.apple.com/en-us/HT202035

**Pull request closed without merging**